### DISCUSSION TITLE
Can't drop files into the App with admin privileges.

### DISCUSSION THREAD

**Discussion by xinghaiyege** [AvaloniaUI/Avalonia#13838] [Q&A] [answered]
Drop files with user privileges work fine, but you cannot drop files with administrator privileges.

||Version|
|-|-|
|OS|Windows 11|
|Avalonia|11.0.5|

**Comment by maxkatz6** [comment, marked as answer]
By design. Window's design.
Windows doesn't allow dropping files to the admin privileges app, unless it was initiated from the same privilege.
Avalonia cannot redefine this behavior.
